docs/turn-howto.md

```diff
@@ -139,7 +170,7 @@ Your home server configuration file needs the following extra keys:
 
 As an example, here is the relevant section of the config file for matrix.org:
```

Comment:
Can we expand this briefly to say that this is for configurations with default port and using TURN (rather than TURNs)?

Reply:
done, though it does rather lead to the question: shouldn't we be using TLS on matrix.org?